File: python-os/fileCompare.py
def FileCompare(liste1, liste2):
    for i in liste1:
        if liste2.count(i) == 0:
            print(f"\ndifferent in path1: {i}", end="")

    print("\n", end="")

    for l in liste2:
        if liste1.count(l) == 0:
            print(f"\ndifferent in path2: {l}", end="")

    print("\n\n", end="")

File: golang-io/fileCompare/fileCompare.go
package fileCompare

import (
	"FileComparator/tools"
	"fmt"
)

func FileCompare(liste1 []string, liste2 []string) {
	for _, i := range liste1 {
		if tools.Count(liste2, i) == 0 {
			fmt.Printf("\ndifferent in path1: %v", i)
		}
	}

	fmt.Print("\n")

	for _, l := range liste2 {
		if tools.Count(liste1, l) == 0 {
			fmt.Printf("\ndifferent in path2: %v", l)
		}
	}

	fmt.Print("\n\n")
}
